lib/log/json_formatter.rb

```diff
@@ -0,0 +1,26 @@
+require 'json'
+
+module Log
+  class JSONFormatter
+
+    def call(severity, time, progname, msg)
+      log_item = {
+        app: progname,
+        host: host,
+        level: severity,
+      }
+      log_item[:message] = msg[:message] if msg[:message]
```

Comment:
I like the idea, but `msg` is a hash that can have multiple things at once. It's completely valid to have an event and corresponding metrics. So I don't see how we could generalize this more.